### DISCUSSION TITLE
Варианты размещения методических рекомендаций

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#251] [Минимальный набор артефактов]
На встрече 5 декабря 2023 обсудили следующие варианты размещения методических рекомендаций по написанию артефактов на fsh:
**Вариант 1. На сайте сообщества Fhir.ru, как мы это делали раньше: https://fhir.ru/support**
Плюсы: эти рекомендации действительно предназначены участникам сообщества, и относятся ко всем руководствам по реализации в целом
**Вариант 2. Включать в конкретные руководства по реализации**

**Comment by Vrodvi** [comment]
Поддерживаю "Вариант 1."

Действительно, методические рекомендации - это рекомендации для авторов руководств, а не для их пользователей, и в этой ключе им не место в руководстве.

Сами создатели FHIR похоже тоже озаботились необходимость подобных методологических рекомендаций, вот наткнулся тут на их подход к этому вопросу - создана отдельная спецификация с рекомендациями авторам руководств, см.:
- https://build.fhir.org/ig/FHIR/ig-guidance/index.html
- https://github.com/FHIR/ig-guidance

**Reply by eugenekogan-spb** [reply]
Добавил приведенные ссылки на страничку https://fhir.ru/support.fhir-guide.
Сами странички рекомендаций перенесем на сайт после их утверждения в дискуссиях.
